src/main/archetype/dispatcher.ams/src/conf.d/proxy/cif-magento-graphql.proxy

```diff
@@ -0,0 +1,9 @@
+
+Include /etc/httpd/conf.d/variables/cif-magento-proxy.vars
```

Comment:
Lets please rename the files to `aem-commerce` or `cif` only, since this can be used for non Magento as well.